cddl/Makefile

```diff
@@ -57,3 +58,9 @@ clean: ; rm -f $(CLEANFILES)
 
 # Extract the CBOR tags defined by CoRIM/CoMID (i.e., those in the 5xx space)
 cbor-tags.txt: $(wildcard *.cddl) ; grep -h '#6\.5' *cddl | sort -u -t'=' -k2 > $@
+
+# Return an error if there are duplicat CBOR Tags defined
+cbor-tags-unique: $(wildcard *.cddl) ; \
+	 grep -h '#6\.5' *cddl | sort -u -t'=' -k2 > cbor-tags.txt ; \
```

Comment:
I guess this target should depend explicitly on `cbor-tags.txt` -- this way L64 becomes redundant